src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -179,8 +179,8 @@ public NetworkPeerState State
         /// <summary>List of node's modules attached to the peer to receive notifications about various events related to the peer.</summary>
         public NetworkPeerBehaviorsCollection Behaviors { get; private set; }
 
-        /// <summary>Information about the peer including its network address, protocol version, time of last contact.</summary>
-        public NetworkAddress PeerAddress { get; private set; }
+        /// <summary>IP address and port on the side of the peer.</summary>
+        public IPEndPoint PeerAddress { get; private set; }
```

Comment:
Can we maybe rename `PeerAddress` to `endPoint` everywhere?

Reply:
I was thinking to completely remove this as we have remote socket endpoint variable above.but when I was about to merge those two I found out that they are not perfectly equivalent.

therefore, I have decided to leave this as peer address.just to avoid confusion.does it work for you?

Reply:
hmm actually you are right it is confusing, i think i can still have those two separated if i name this PeerEndPoint, instead of just EndPoint. So, would PeerEndPoint work for you?

Reply:
`PeerEndPoint` is much better yes!